src/index.html

```diff
@@ -9,8 +9,8 @@
 
     <link
```

Comment:
Same above, I think it does not need to be attached

Reply:
fix: I connected jquery through a gulp